lib/ldclient-rb/ldclient.rb

```diff
@@ -247,8 +247,19 @@ def match_target?(target, user)
 
     end
 
+    def match_user?(variation, user)
+      if !!variation[:userTarget]
+        return match_target?(variation[:userTarget], user)
+      end
+      return false
+    end
+
     def match_variation?(variation, user)
```

Comment:
Since the `userTarget` is a part of the variation, this function should probably be named `matchTargets` or something, since it a user may match a given variation, but this function would return false.